Formula/mockolo.rb

```diff
@@ -0,0 +1,46 @@
+class Mockolo < Formula
+  desc "Efficient Mock Generator for Swift"
+  homepage "https://github.com/uber/mockolo"
+  url "https://github.com/uber/mockolo/archive/1.1.5.tar.gz"
+  sha256 "53c828299311eb315b096e4d687811da49c0a45ab9647535c2879c144b016bbc"
+
+  depends_on :xcode => "11.0"
```

Comment:
Is this a runtime dependency as well?

Reply:
@elsh I believe we only need this to build mockolo correct?

Reply:
Yes, it's a build dependency